### PR TITLE
Feature add and remove method

### PR DESCRIPTION
**Pull request by aimemalaika**
## What does this PR do?
This PR is raised to create and initialize the todo list project
## Description of the task to be completed
- Added add task methods
- Added remove task methods
- Added update task method
## How should it be tested
- clone the repository\
  `git clone https://github.com/aimemalaika/todo-list.git`
- navigate to the folder\
  `cd todo-list`
- checkout the feature\
  `feature-add-and-remove-method`
- Install packages\
    `npm install`
- Run the app\
    `npm start`
- Create build app\
    `npm run build`
- Test Linters app\
    `npm test`

### CODE REVIEW

File: src/Components/Tasks.js [hunk 1]
```diff
@@ -1,64 +1,67 @@
...
+    window.localStorage.setItem('tasks', JSON.stringify(result));
+    const count = this.load();
+    document.querySelector('.number-label').innerText = count;
+  }
+
+  static updateData(index, value) {
+    const data = this.db();
+    data[index].descption = value;
+    window.localStorage.setItem('tasks', JSON.stringify(data));
   }
```
Comment (mikethreels) [inline src/Components/Tasks.js:55]: - [x] when updating a task the changes are not saved to the local storage, therefore after reloading the page it changes back to the original task description.

File: src/Components/Tasks.js [hunk 2]
```diff
@@ -1,64 +1,67 @@
...
 
   static createElement = (task) => {
     const listItem = document.createElement('li');
+    const form = document.createElement('form');
+    form.classList.add('current-task');
     listItem.classList.add('dragable');
     const button = document.createElement('p');
     const input = document.createElement('input');
+    input.setAttribute('data-task-id', task.index);
```
Comment (mikethreels) [inline src/Components/Tasks.js:71]: - [ ] when clicking the checkbox the task completion status doesn't change in the local storage and therefore after reloading the status changes back to what it originally was.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Aime, 

Good job creating the add and delete function they are working as expected 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/Tasks.js (hunk 2 above)
Comment (aimemalaika) [reply src/Components/Tasks.js:71]: Hello @mikethreels, This is for the next pull request.
Screen Shot 2022-01-14 at 12 41 35
check the project requirements

**Review by moise10r — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @aimemalaika ** , 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Your App works as expected, keep rocking :rocket: 


Cheers and Happy coding!👏👏👏💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**